src/libs/actions/Policy.ts

```diff
@@ -952,27 +1020,30 @@ function updateWorkspaceCustomUnitAndRate(policyID, currentCustomUnit, newCustom
         },
     ];
 
-    const newCustomUnitParam = _.clone(newCustomUnit);
-    newCustomUnitParam.rates = _.omit(newCustomUnitParam.rates, ['pendingAction', 'errors']);
-    API.write(
-        'UpdateWorkspaceCustomUnitAndRate',
-        {
-            policyID,
-            lastModified,
-            customUnit: JSON.stringify(newCustomUnitParam),
-            customUnitRate: JSON.stringify(newCustomUnitParam.rates),
-        },
-        {optimisticData, successData, failureData},
-    );
+    const newCustomUnitParam = lodashClone(newCustomUnit);
+    newCustomUnitParam.rates = {...newCustomUnitParam.rates, pendingAction: undefined, errors: undefined};
```

Comment:
I guess it should be more similar to the omit functionality, wdyt?
```suggestion
    const {pendingAction, errors, ...newRates} =newCustomUnitParam.rates ?? {};
    newCustomUnitParam.rates = newRates;
```

Reply:
Agreed, that is clearer.